auth-web/src/store/modules/org.ts

```diff
@@ -97,6 +97,7 @@ export default class OrgModule extends VuexModule {
     await this.context.dispatch('syncActiveOrgMembers')
     await this.context.dispatch('syncPendingOrgMembers')
     await this.context.dispatch('syncPendingOrgInvitations')
+    await this.context.dispatch('business/syncBusinesses', null, { root: true })
```

Comment:
I believe this change will fix https://github.com/bcgov/entity/issues/2943.  If it does, can you move that ticket to in review at the same time and tag this PR with the ticket #? :)